test/config/config.ts

```diff
@@ -14,60 +16,97 @@ export const buildAirnodeConfig = () => ({
       },
       type: 'evm',
       options: {
-        txType: 'eip1559',
-        baseFeeMultiplier: 2,
-        priorityFee: {
-          value: 3.12,
-          unit: 'gwei',
-        },
+        fulfillmentGasLimit: 500000,
+        gasPriceOracle: [
```

Comment:
We could also use `latestBlockPercentileGasPrice` and `providerRecommendedGasPrice` strategy configs here to make sure the E2E tests use them as well :thinking: 

It shouldn't affect the tests so just adding the strategies in the array should be fine, though